### PR TITLE
Added Create Stream Endpoint

### PR DESCRIPTION
**Pull request by vignesha22**

### CODE REVIEW

File: examples/package.json [hunk 1]
```diff
@@ -26,7 +26,6 @@
     "18-cross-chain-quote": "../node_modules/.bin/ts-node ./src/18-cross-chain-quote.ts"
   },
   "dependencies": {
-    "dotenv": "^16.0.1",
-    "@superfluid-finance/sdk-core": "0.4.2"
+    "dotenv": "^16.0.1"
```
Comment (zincoshine) [inline examples/package.json:29]: Pls use a specific version of the package.

File: test.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+// import { ethers, BigNumber } from "ethers";
+
+
+async function main() {
+    try {
+        // let provider = new ethers.providers.JsonRpcProvider('https://rinkeby.infura.io/v3/502a8e83d34d471596ca857eec9771ff')
+        // const provider = new ethers.providers.JsonRpcProvider('https://rpc-mumbai.maticvigil.com')
+        const sdk = new Sdk(
+        // "0x513a984bbd054d9fb6d8ba656183185f55bad24a8f900a57a820077374fa9779",
+            "0x9648a8add89fd006b4d5a8f913b6547ffab680fa3b0bcd91247cc23092e47fd0",
```
Comment (zincoshine) [inline test.ts:12]: Is this file relevant? Why is it part of the source code?

**Review by zincoshine — CHANGES_REQUESTED** [review]

File: test.ts (hunk 2 above)
Comment (vignesha22) [reply test.ts]: no just used for testing

**Review by zincoshine — APPROVED** [review]

**Review by marie-fourier — APPROVED** [review]
lgtm

**Review by riccardopillar — APPROVED** [review]

**Pull request merged**